src/cnc/models/config/service.py

```diff
@@ -481,7 +499,7 @@ def cli_info(self):
             "type": self.settings.type,
         }
 
-        if self.is_frontend or self.is_backend:
+        if self.is_frontend or self.is_backend or self.is_web:
```

Comment:
After the update to `is_web` this can be simplified:
```python
if self.is_web:
```

Reply:
Modifications Reference: https://github.com/coherenceplatform/cnc/pull/90/commits/7e62798906dcb1450236260512a3de5636f93152